### PR TITLE
fix(overlay): fixed undefined error on web

### PR DESCRIPTION
**Pull request by Calinteodor**
Untitled

### CODE REVIEW

File: react/features/overlay/middleware.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import {
 } from '../base/lib-jitsi-meet/functions.any';
 import StateListenerRegistry from '../base/redux/StateListenerRegistry';
 
-import { openPageReloadDialog } from './actions';
+import { openPageReloadDialog } from './actions.native';
```
Comment (damencho) [inline react/features/overlay/middleware.ts:11]: Isn't this mixing native into the web?

Comment (Calinteodor) [reply react/features/overlay/middleware.ts:11]: This middleware is now part of middlewares.native. Before it was part of middlewares.any.

Comment (damencho) [reply react/features/overlay/middleware.ts:11]: But then shouldn't this be middleware.native then?

Comment (Calinteodor) [reply react/features/overlay/middleware.ts:11]: I can add an .native extension to the file to be more specific and avoid confusion, yes.

Comment (damencho) [reply react/features/overlay/middleware.ts:11]: 👍

**Review by damencho — APPROVED** [review]

**Review by saghul — APPROVED** [review]

**Pull request merged**